core/src/main/java/org/mapfish/print/VersionInfo.java

```diff
@@ -0,0 +1,72 @@
+package org.mapfish.print;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.annotation.Nullable;
+import javax.annotation.PostConstruct;
+import javax.servlet.ServletContext;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+
+/**
+ * Allows to access version information (from the manifest).
+ *
+ * Works only in a servlet context.
+ */
+public class VersionInfo {
+    private static final Logger LOGGER = LoggerFactory.getLogger(VersionInfo.class);
+
+    @Nullable
+    public static final String MAPFISH_PRINT_VERSION = "Mapfish-Print-Version";
+    public static final String GIT_VERSION = "Git-Revision";
+
+    private Attributes attrs;
+
+    @Autowired
+    private ServletContext servletContext;
+
+    @PostConstruct
+    public final void init() {
+        attrs = getAttributes();
+        LOGGER.warn("Starting print version {} ({})", getVersion(), getGitHash());
```

Comment:
Yes, INFO is not shown on production machines and since this info is needed on production machine, I've put it at WARN level. I could put a special logger config for this class, but since all the c2cgeoportal projects have a custom logger configuration file, it would mean to change it for all the projects. So I went the easy route and artificially increased the level of this message.